### PR TITLE
Add target type radio, autodetect emulator support and dynamic target dr...

### PR DESCRIPTION
**Pull request by kflores772**
...opdown

### CODE REVIEW

**Comment by kflores772** [comment]
This is still in progress

**Comment by rcruz** [comment]
Missing lib/ui/views/targetSelector.js

**Comment by rcruz** [comment]
Looks good so far. Just need to run "emulate" instead of "run" when simulator is detected

**Comment by kflores772** [comment]
The latest commit leads to an error since the routes no longer call the webworks CLI (11c339501958c3db8826e4763bdd1806fd02c4bc). Instead, the route attempts to run Project 1/platforms/blackberry10/cordova/emulate which does not exist.

File: lib/ui/models/project.js [hunk 1]
```diff
@@ -89,12 +89,18 @@ Project = Backbone.Model.extend({
         }, callback, callback);
     },
     install: function (options, callback) {
+        if (!options.device) {
+            options.debug = true;
```
Comment (rcruz) [inline lib/ui/models/project.js:93]: Not sure if you need to force debug mode when using a simulator

**Comment by rcruz** [comment]
Target filter works correctly. Deploy to simulator works as well.

**Comment by rcruz** [comment]
r+, merging to v2.1.0

**Pull request closed without merging**